playbooks/ORCHESTRATION.md

```diff
@@ -0,0 +1,219 @@
+# Orchestration
+
+This playbook describes how to orchestrate a root signing event.
+
+## Pre-work
+
+1. Check the [calendar](https://calendar.google.com/calendar/u/0?cid=Y19ydjIxcDJuMzJsbmJoYW5uaXFwOXIzNTJtb0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) for upcoming root signing events. 
+
+2. Make updates to the targets and delegation configuration files, see [configuration](#targets-and-delegation-configuration).
+
+3. Double-check the configured [role expirations](https://github.com/sigstore/root-signing/blob/e3f1fe5e487984f525afc81ac77fa5ce39737d0f/cmd/tuf/app/init.go#L28).
```

Comment:
Not for docs, but can this be a part of a pre-verification step somewhere in the scripts?

Reply:
Adding this here to track for v5: https://github.com/sigstore/root-signing/issues/98

But I might be misunderstanding what you mean.